imap_processing/lo/l1a/lo_data_container.py

```diff
@@ -0,0 +1,66 @@
+"""Container for IMAP-Lo L0 data classes."""
+
+from imap_processing.lo.l0.lo_apid import LoAPID
+
+
+# TODO: Not sure if this class is worth having.
+class LoContainer:
+    """IMAP-Lo Level 0 data container class.
+
+    The IMAP-Lo Level 0 data container class is used to store
+    data for all 19 IMAP-Lo L0 unpacked data to allow filtering
+    by APID.
+
+    Attributes
+    ----------
+    data : dict
+        dictionary of IMAP-Lo data classes for L0
+
+
+    Methods
+    -------
+    add():
+        Adds another data class to the container
+
+    filter_apid():
+        Filters the container and returns a list of
+        data classes for the APID
+    """
+
+    data_dict: dict
```

Comment:
What does this line do? I'm not sure i've seen something like this before. Is it a way to ensure the `data_dict` attribute remains a `dict`?

Reply:
I actually need to remove this since I changed this to not be a data class. But for data classes if you define the variables it will automatically add them as inputs to the __init__ function without you adding an __init__ function.